packages/protocol/contracts/common/MetaTransactionWallet.sol

```diff
@@ -48,7 +48,7 @@ contract MetaTransactionWallet is
    * @return The storage, major, minor, and patch version of the contract.
    */
   function getVersionNumber() public pure returns (uint256, uint256, uint256, uint256) {
-    return (1, 1, 0, 0);
+    return (1, 1, 0, 1);
```

Comment:
Not sure why this is still not up to date. It would seem this latest commit should've required a updated patch version, but it passed the build without it https://github.com/celo-org/celo-monorepo/commit/009292124bf941d73f85fea2a354394a322874e9

Reply:
So when that build ran, I did not yet have cut the branch which would have caused it to compare against the release branch

Reply:
either way, it clearly changed the bytecode, so I'd think it would want to be bumped??

Reply:
Yes bumped make sense!